src/Akka.Persistence.Azure.Tests/Query/AzureTableCurrentEventsByTagSpec.cs

```diff
@@ -32,5 +39,67 @@ public AzureTableCurrentEventsByTagSpec(ITestOutputHelper output)
         }
 
         public static string TableName { get; private set; }
+
+        [Fact(DisplayName = "Read journal should be able to retrieve all tagged persisted messages beyond the first 1000 messages")]
+        public async Task RetrieveAllTaggedRows()
+        {
+            if (ReadJournal is not ICurrentEventsByTagQuery queries)
+                throw IsTypeException.ForMismatchedType(nameof(ICurrentEventsByTagQuery), ReadJournal?.GetType().Name ?? "null");
+
+            var allMessages = Enumerable.Range(0, 2000)
+                .Select(i => $"green {i}").ToArray();
+            var a = Sys.ActorOf(TestPersistenceActor.Props("a"));
+            foreach (var str in allMessages)
+            {
+                a.Tell(str);
+                ExpectMsg($"{str}-done");
```

Comment:
Do a `ReceiveN` outside of the loop instead so we don't have to await each operation individually - will make the test run significantly faster.

Reply:
done